pipeline-maven/src/main/java/org/jenkinsci/plugins/pipeline/maven/MavenPublisherStrategy.java

```diff
@@ -71,7 +72,18 @@ public List<MavenPublisher> buildPublishersList(
                     Jenkins.get().getDescriptorList(MavenPublisher.class);
             for (Descriptor<MavenPublisher> descriptor : descriptorList) {
                 try {
-                    defaultPublishersById.put(descriptor.getId(), descriptor.clazz.newInstance());
+                    if (!JacocoReportPublisher.DescriptorImpl.class.isAssignableFrom(descriptor.getClass())) {
```

Comment:
By default, the implicit publisher strategy would add the a `JacocoReportPublisher` and it would fail the build.
So that we don't break a build because we are not loading the Jacoco reports, I'm simply marking in the build log the alternative.

Reply:
Ok but why this inverted if ?

You want to do a special case for the Jacoco publisher, it should be on the if, and the else the default case no ?

Reply:
Even if I tend to agree in most case, I don't like to use `if (X) { continue; } else { doSomething(); }`. I find it easier to read to revert the branch condition.

Reply:
But you are absolutely right. I just inverted the condition in https://github.com/jenkinsci/pipeline-maven-plugin/pull/881/commits/36df27db1c2c4aa850c4342cdf0df90c8c03ee08

Reply:
I got your point, thanks
I am just thinking that this `if` is here for some time, but will disappear soon (I hope)

By the way, was just a question, not a hard requirement to change ;)

Reply:
No no but you had a great point and was an easy "fix". 
I first wrote the if with negative testing because I only added the log in a second time. I should have inverted the test when I added the log message really.